.github/workflows/ci.yml

```diff
@@ -44,21 +63,19 @@ jobs:
 
       - name: Build wheels
         run: |
-          docker run -v $PWD:/root -w /root quay.io/pypa/manylinux1_x86_64 bash /root/bindings/python/tools/build_wheel.sh
-
-      - name: Install dependencies
-        run: |
-          python -m pip install wheelhouse/*36*.whl
-          python -m pip install pytest
-
-      - name: Test with pytest
-        run: |
-          python -m pytest bindings/python/test/test.py
+          python -m pip install cibuildwheel==1.7.2
+          python -m cibuildwheel bindings/python --output-dir wheelhouse
+        env:
+          CIBW_BEFORE_ALL: bindings/python/tools/prepare_build_environment.sh
+          CIBW_BEFORE_BUILD: pip install pybind11==2.6.0
+          CIBW_TEST_COMMAND: pytest {project}/bindings/python/test/test.py
+          CIBW_TEST_REQUIRES: pytest
+          CIBW_SKIP: cp27-* pp* *i686
```

Comment:
Previous versions of the Tokenizer used `manylinux1` but it looks like `cibuildwheel` uses `manylinux2010` by default. Can we try building under `manylinux1` to not require users to upgrade their pip version? It seems the following should be enough:

```yaml
CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
```